### PR TITLE
Table - Fixed stickyHeader option

### PR DESCRIPTION
**Pull request by tonynd**
### Summary
- Fix stickyHeader option for table. This prevents the header from sticky to the top of the browser window when scrolling up and down the table.
- `stickyHeader` option now default to `false`.

### Review Notes/ Questions/ TODOs
- A change to how the headers stickiness is handled so broke the stickyHeader option.

### CODE REVIEW

File: table/README.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ A table component with checkboxes, sticky head, sticky right panel, and horizont
 
 # Change Log
 
+##### 10.0.6
+- Fixed `stickyHeader` option for table.
```
Comment (ben-burlingham) [inline table/README.md]: Can kill this line, the next one is descriptive and concise

File: table/src/Table.jsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ export default class extends React.Component {
     showCheckboxes: false,
     showSidebar: false,
     sortState: { direction: SORT_DIRECTION.NONE, attribute: null },
-    stickyHeader: true,
+    stickyHeader: false,
```
Comment (dancingshell) [inline table/src/Table.jsx]: this seems like a breaking change/ major version bump. is there a reason you needed to switch this?

File: table/README.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ A table component with checkboxes, sticky head, sticky right panel, and horizont
 
 # Change Log
 
+##### 10.0.6
```
Comment (ben-burlingham) [inline table/README.md]: Unsticky headers will affect many teams, this is a breaking change so should be Major

File: table/src/Table.jsx [hunk 4]
```diff
@@ -100,10 +100,10 @@ export default class extends React.Component {
 
   componentDidMount() {
     window.addEventListener('table/expandAll', this.onExpandAll);
-    window.addEventListener('scroll', this.onScroll);
+    this.props.stickyHeader && window.addEventListener('scroll', this.onScroll);
```
Comment (ben-burlingham) [inline table/src/Table.jsx]: Nice

File: table/src/Table.jsx (hunk 2 above)
Comment (tonynd) [reply table/src/Table.jsx]: Talked to @ben-burlingham about this. Would like to keep the consistency of defaultProps in terms of boolean being `false`.

**Comment by ben-burlingham** [comment]
Also can you indicate _Breaking_ in the README - this is something we're doing to help people during their updates, so they can target fixes faster.  Something like:
> 11.0.0
> - _Breaking_ Changing `stickyHeader` to be `false` by default

**Pull request merged**